glidertest/tools.py

```diff
@@ -194,51 +194,55 @@ def plot_basic_vars(ds, v_res=1, start_prof=0, end_prof=-1):
     [a.set_ylim(depthG.max() + 10, -5) for a in ax]
     [a.grid() for a in ax]
 
-
-def chl_first_check(ds):
+def optics_first_check(ds, var='CHLA'):
     """
-    Function to asses any drift in deep chlorophyll data and the presence of any possible negative data
+    Function to asses any drift in deep optics data and the presence of any possible negative data
     This function returns plots and text
     """
     

```

Comment:
How about a check here, so that if the requestsed var is not in the ds it prints an error and returns?
```python
if var not is ds.variables:
    print("oh no")
    return
```

You could also add this to the tests